iPetCare/Application/Dtos/Examinations/ExaminationsCreateExaminationDtoRequest.cs

```diff
@@ -9,7 +9,8 @@ public class ExaminationsCreateExaminationDtoRequest
         [Required]
         public DateTime Date { get; set; }
         public int ExaminationTypeId { get; set; }
-        public string NoteId { get; set; }
+        public Guid? NoteId { get; set; }
+        [Required]
         public string PetId { get; set; }
```

Comment:
PetId też jest Guidem :D

Reply:
Fuck musiałem przegapić, dobrze że to sprawdzasz :D